apps/kotlin-gradle/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+	id("org.springframework.boot") version "2.1.8.RELEASE"
```

Comment:
i wonder about version ranges here again.

Reply:
I researched it and couldn't find any way to do it. I experimented as well to no avail. I'm going with it's not possible at the moment but maybe someone else could find a solution.

Reply:
It's fine if we can't. But we'll have to try and stay on top of updating it (I'm having flashbacks to the Struts example repos that used versions with the Equifax CVE for months after it went public).